webapp/src/components/kanban/kanban.tsx

```diff
@@ -275,9 +279,7 @@ const Kanban = (props: Props) => {
                                 key={card.id}
                                 readonly={props.readonly}
                                 isSelected={props.selectedCardIds.includes(card.id)}
-                                onClick={(e) => {
-                                    props.onCardClicked(e, card)
-                                }}
+                                onClick={props.onCardClicked}
```

Comment:
I change the component Kanban card component to fit in the signature of the `props.onCardClicked`